src/defaultSettings.js

```diff
@@ -19,6 +11,14 @@ export default {
             rotate_tick_labels: true,
             vertical_space: 75
         },
+        {
+            value_col: 'DY',
+            type: 'linear',
+            order: null,
+            label: 'Study Day',
+            rotate_tick_labels: false,
+            vertical_space: 0
+        },
         {
```

Comment:
Guessing we're going to remove numeric visit axis in a later PR here ...

Reply:
Yup.